src/firmware/Firmware-IGVC-2018/Firmware-IGVC-2018.ino

```diff
@@ -0,0 +1,146 @@
+/* Created May 5th by Chris Heathe
+ *  
+ * Basic arduino motor driver that subsribes to the /cmd_vel topic 
+ * and prints corresponding servo values to the left and write motors 
+ * of Elsa.
+ * 
+ * PID loop has now been integrated, Kp, Ki, Kd values will need to be calibrated in physical testing
+ * The library that was used can be found at:
+ * 
+ * https://playground.arduino.cc/Code/PIDLibrary
+ * 
+ * !!NOTES ON USING THE ros.h ARDUINO LIBRARY!!
+ * 
+ * Install the 'ros-kinetic-rosserial-server' package
```

Comment:
You can automatically do this by adding `<run_depend>rosserial_server</run_depend>` to `package.xml`